### ISSUE TITLE
Formal documentation

### ISSUE THREAD

**Issue by arya-tama** [cdcseacave/openMVS#653]
Hi,

So far, I have to the read the source or running binaries with help command to understand how the pipeline works.
I wonder if there is any formal documentation like a paper? That would be helpful. Anyway, thanks for a great work! This helps my master thesis so much.

**Comment by cdcseacave** [comment]
didn't the wiki help you?

**Comment by arya-tama** [comment]
Yes, it helps. Maybe with some explanations like the theory, flow chart, math and so on would be better.

**Comment by Ryujinosorus** [comment]
Hey,

Did you find some paper ? 

In case if you haven't find.

DensifyPointCloud  is based on the Patch-Match algorithm

You can have more info on mesh reconstruction here : Exploiting Visibility Information in Surface Reconstruction to Preserve Weakly Supported Surfaces M. Jancosek et al. 2014.

Mesh refinement : High Accuracy and Visibility-Consistent Dense Multiview Stereo HH. Vu et al. 2012.

Mesh Texturing : Let There Be Color! - Large-Scale Texturing of 3D Reconstructions M. Waechter et al. 2014.

If you found any more, can you share them here?

**Comment by arya-tama** [comment]
Hi,

I only found the paper about that Large-Scale Texturing of 3D Reconstructions. So thank you very much for your share! I'll let you know if I get another information

**Comment by cdcseacave** [comment]
Closing as stale. The codebase has been substantially updated on the \develop\ branch. Please reopen with reproduction steps if the issue persists on the latest code.
